questionpy_sdk/commands/_helper.py

```diff
@@ -39,23 +39,17 @@ def infer_package_kind(pkg_string: str) -> PackageLocation:
     pkg_path = Path(pkg_string)
 
     if pkg_path.is_dir():
+        # Always rebuild package.
+        build_dir_package(pkg_path)
```

Comment:
davor noch ein `shutil.rmtree` aufs dist-dir?

Reply:
Gehört das nicht eher in build_dir_package oder DirPackageBuilder?

Reply:
> Gehört das nicht eher in build_dir_package oder DirPackageBuilder?

Ja, richtig. Vielleicht einfach in `build_dir_package`.

Reply:
Lösung: Später richtig machen... #107